src/simulate.jl

```diff
@@ -20,8 +20,12 @@ all the corresponding data.
 
 `parameters` contains all the simulation information, including the number of
 replications, the type of simulation, the hydrological years to sample from, etc.
+
+### Keyword Arguments
+`async` is a boolean that sets whether SDDP.jl will run in parallel. Currently
+recommended to be set to false.
 """
-function simulate(JADEmodel::JADEModel, parameters::JADESimulation)
+function simulate(JADEmodel::JADEModel, parameters::JADESimulation; async::Bool=false)
     d = JADEmodel.d
     sddpm = JADEmodel.sddpm
```

Comment:
You'll note by now that much of JADE could be refactored into smaller functions :smile:

But let's just define the parallel scheme once up front:
```suggestion
    sddpm = JADEmodel.sddpm
    parallel_scheme = async ? SDDP.Threaded() : SDDP.Serial()
```

Reply:
I'd go for defining it once up front yep